packages/peregrine/lib/Price/__tests__/__snapshots__/price.spec.js.snap

```diff
@@ -48,7 +48,7 @@ Array [
     1
   </span>,
   <span>
-     
+     
```

Comment:
Why do I feel like I've seen this several times now; any idea why the whitespace in this snap keeps changing?

Reply:
It keeps sneaking in every time I update the mocks. Thanks for catching it, Tommy. I'll have to research and fix this once and for all. Even @jcalcaben has this issue.